src/org/opendatakit/briefcase/ui/ExportPanel.java

```diff
@@ -577,6 +577,7 @@ public void successfulCompletion(ExportSucceededEvent event) {
     lblExporting.setText("SUCCEEDED!");
     setActiveExportState(false);
   }
+
   @EventSubscriber(eventClass = ExportSucceededWithErrorsEvent.class)
   public void successfulCompletionWithErrors(ExportSucceededWithErrorsEvent event) {
     exportStatusList.append("\n").append("SUCCEEDED, BUT WITH ERRORS!");
```

Comment:
I went back and forth on this. I think what we should probably do is file an issue to replace the all caps messages in a single PR.